index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>tribute-page</title>
+ <link rel="stylesheet" href="style.css">
+</head>
+<body>
+ <main id="main">
+  <h1 id="title">
+   Omar Suleiman
+  </h1>
+  <p id="extra"> American Islamic scholar </p>
+  <figure id="img-div">
+   <img id="image" src="https://th.bing.com/th/id/R.1d4ab77eb8fbd81068b50c2b0edd4e74?rik=6Z2gv2%2bBl6rm5g&pid=ImgRaw&r=0" alt="Omar Suleiman 's photo">
+  <br>
+   <figcaption id="img-caption">
+   Dr. Omar Suleiman is the President of the Yaqeen Institute for Islamic Research and a professor of Islamic Studies at Southern Methodist University
+  </figcaption>
+  </figure>
+  <p id="tribute-info">
+   Omar Suleiman (born June 3, 1986) is an American Islamic scholar and civil rights activist. He is the founding president of the Yaqeen Institute for Islamic Research and an adjunct professor of Islamic studies and member of the Ethics Center Advisory Board at Southern Methodist University.
+  </p>
+ <br>
+ <p id="tribute-info">
+  Omar Suleiman was born in 1986 in New Orleans to a family of Palestinian Muslims. He holds bachelor's degrees in accounting and Islamic law and Master's degrees in Islamic finance and political history. He completed doctoral studies in Islamic thought and civilization at the International Islamic University Malaysia
+ </p>
+ <br>
+ <p id="tribute-info">After completing his doctoral degree in Malaysia, Suleiman returned to New Orleans and served as the imam of the Jefferson Muslim Association for six years, and as director of the "Muslims for Humanity" Hurricane Katrina relief effort in late 2005. He co-founded the East Jefferson Interfaith Clergy Association and received an award for outstanding civic achievement from the mayor and city council of New Orleans in 2010.
+ </p>
+ <br>
+ <p id="tribute-info">In 2016, Suleiman founded the Muslim think tank Yaqeen Institute for Islamic Research. He is also the founding director of MUHSEN (Muslims Understanding and Helping Special Education Needs), a nonprofit umbrella organization that aims to create more inclusive Muslim communities that better cater to the disabled and their families.</p>
+ <br>
+<p id="tribute-info">He was (as of May 2019) Adjunct Professor of Islamic Studies at Southern Methodist University, where he was also a member of the Maguire Ethics Center Advisory Board.</p>
+<br>
+<p id="tribute-info">He is (as of September 2022) Resident Scholar of the Valley Ranch Islamic Center and co-chair of Faith Forward Dallas at Thanks-Giving Square.</p>
+<p id="extra">If you have time, you should read more about this incredible human being on his <a id="tribute-link" target="_blank" rel="noopener" href="https://www.facebook.com/imamomarsuleiman/">Facebook</a>.</p>
+</main>
+</body>
```

Comment:
Your codes are not alligned correctly and are not clean try to make their spacing right.
- [**Suggession**] - On your HTML file, at the bottom right side of your vscode click on `Spaces`  then click on `Index using spaces`, and then select `2` from there. Then align your components according to the 2 spaces.
- Example:
```
<div>
  <p>...</p>
  <h1>.... </h1>
  <img ... />
</div>
```

Reply:
@ahzamir Thank you for your comment. I will go further and add some more styles to the page.